app/src/components/TimestampOrderBookScroller.js

```diff
@@ -1,17 +1,28 @@
 import React, { Component, createRef } from 'react';
 import { withStyles } from '@material-ui/core/styles';
 
-import { Button, Box } from '@material-ui/core';
+import {
+    Button, Box, Typography, ButtonGroup,
+} from '@material-ui/core';
+import ChevronLeftSharpIcon from '@material-ui/icons/ChevronLeftSharp';
+import ChevronRightSharpIcon from '@material-ui/icons/ChevronRightSharp';
 import { Styles } from '../styles/TimestampOrderBookScroller';
 import MultiDirectionalScroll from './MultiDirectionalScroll';
 import PriceLevel from './PriceLevel';
 import { getOrderBookListItemsAsArray, listItemsEquals } from '../utils/order-book-utils';
 
+import { LEFT_ARROW_KEY_CODE, RIGHT_ARROW_KEY_CODE, SNAPSHOT_INSTRUMENT } from '../constants/Constants';
+import OrderBookService from '../services/OrderBookService';
+
 const MIN_PERCENTAGE_FACTOR_FOR_BOX_SPACE = 0.35;
 
 class TimestampOrderBookScroller extends Component {
     middleReferenceItem = null;
 
+    componentDidMount() {
+        window.addEventListener('keyup', this.onKeyUp);
```

Comment:
might want to remove this eventListener once this components unmounts. Or else it won't find the `onKeyUp` function if the component isn't displayed anymore and the user presses the right, left arrow keys